blocks/header/header.css

```diff
@@ -273,3 +312,59 @@ header nav .nav-sections ul > li > ul > li {
 header nav .nav-tools {
   grid-area: tools;
 }
+
+.languagenavigation {
+  width: 10%;
+}
+
+.cmp-languagenavigation__item--active {
```

Comment:
these old class names (ie `.cmp-languagenavigation__item--active`) do not exist in Franklin. I don't see how these will work in anyway